modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -106,11 +105,7 @@ For information on index storage, see xref:learn:services-and-indexes/indexes/st
 
 _All_ auto-compaction parameters can be edited, following bucket creation.
 
-The Auto-compaction parameter group contains the following:
-
-** Parameters that are valid for Couchbase Server Enterprise and Community Editions alike; these being xref:rest-api:rest-bucket-create.adoc#autocompactiondefined[autoCompactionDefined], xref:rest-api:rest-bucket-create.adoc#paralleldbandviewcompaction[parallelDBAndViewCompaction], xref:rest-api:rest-bucket-create.adoc#databasefragmentationthresholdpercentage[databaseFragmentationThreshold[percentage&#93;], xref:rest-api:rest-bucket-create.adoc#databasefragmentationthresholdsize[databaseFragmentationThreshold[size&#93;], xref:rest-api:rest-bucket-create.adoc#viewfragmentationthresholdpercentage[viewFragmentationThreshold[percentage&#93;], xref:rest-api:rest-bucket-create.adoc#viewfragmentationthresholdsize[viewFragmentationThreshold[size&#93;], xref:rest-api:rest-bucket-create.adoc#purgeinterval[purgeInterval], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodfromhour[allowedTimePeriod[fromHour&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodfromminute[allowedTimePeriod[fromMinute&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodtohour[allowedTimePeriod[toHour&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodtominute[allowedTimePeriod[toMinute&#93;], and xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodabortoutside[allowedTimePeriod[abortOutside&#93;].
-
-** A single parameter that is valid for Couchbase Server Community Edition only; this being xref:rest-api:rest-bucket-create.adoc#indexcompactionmode[indexCompactionMode].
+The Auto-compaction parameter group contains the following: xref:rest-api:rest-bucket-create.adoc#autocompactiondefined[autoCompactionDefined], xref:rest-api:rest-bucket-create.adoc#paralleldbandviewcompaction[parallelDBAndViewCompaction], xref:rest-api:rest-bucket-create.adoc#databasefragmentationthresholdpercentage[databaseFragmentationThreshold[percentage&#93;], xref:rest-api:rest-bucket-create.adoc#databasefragmentationthresholdsize[databaseFragmentationThreshold[size&#93;], xref:rest-api:rest-bucket-create.adoc#viewfragmentationthresholdpercentage[viewFragmentationThreshold[percentage&#93;], xref:rest-api:rest-bucket-create.adoc#viewfragmentationthresholdsize[viewFragmentationThreshold[size&#93;], xref:rest-api:rest-bucket-create.adoc#purgeinterval[purgeInterval], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodfromhour[allowedTimePeriod[fromHour&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodfromminute[allowedTimePeriod[fromMinute&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodtohour[allowedTimePeriod[toHour&#93;], xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodtominute[allowedTimePeriod[toMinute&#93;], and xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodabortoutside[allowedTimePeriod[abortOutside&#93;].
```

Comment:
Worth noting for the future:

If you want to use square brackets as part of the link text, rather than doing this:

`xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodabortoutside[allowedTimePeriod[abortOutside&#93;]`

It's better to just escape the text like this:

`xref:rest-api:rest-bucket-create.adoc#allowedtimeperiodabortoutside[++allowedTimePeriod[abortOutside]++]`

Reply:
Cheers, thanks!